airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -87,7 +87,8 @@ def __init__(
         super().__init__(http_conn_id=livy_conn_id)
         self.extra_headers = extra_headers or {}
         self.extra_options = extra_options or {}
-        self.auth_type = auth_type or self.auth_type
+        if auth_type:
+            self.auth_type = auth_type
```

Comment:
I needed to adjust it, since it would always set the default `HTTPBasicAuth` from the parent class property and then complain about the lack of `username` and `password` to initialize it.